src/js/views/Planet.js

```diff
@@ -305,15 +523,15 @@ class Planet {
   randomizeUqm() {
     this.randomize(1);
     this.uqmPlanetSeedChoice = this.seedString;
-
+    
     if (this.uqmPlanetSeedChoiceControl != null) {
       this.uqmPlanetSeedChoiceControl.updateDisplay();
     }
   }
 
   pickPlanetType() {
     let typeString = this.uqmPlanetType;
-    let type = this.uqmPlanetTable.findPlanetTypeByName(this.uqmPlanetType);
+    let type = this.UQM_PLANETTABLE.find(m => m.type == this.uqmPlanetType);
```

Comment:
Shouldn't this be `let type = UqmGenerationTable.findPlanetTypeByName(this.uqmPlanetType);`?